builder/builder-next/exporter/snapshotexporter/snapshotexporter.go

```diff
@@ -0,0 +1,27 @@
+package snapshotexporter
+
+import (
+	"context"
+	"strings"
+
+	mobyexporter "github.com/docker/docker/builder/builder-next/exporter"
+	"github.com/moby/buildkit/exporter"
+)
+
+type snapshotExporter struct {
```

Comment:
This name is very confusing. Not sure if it is needed at all (see below). Otherwise, the name would be `containerdImageExporterWrapper`